src/views.py

```diff
@@ -67,4 +68,64 @@ def get_all(ticker):
         with open('data.json', 'w') as stock_json:
             json.dump(new_data, stock_json)
 
-        return new_data
\ No newline at end of file
+        return new_data
+
+
+@app.route('/<ticker>/historical/5_days')
+def get_historical_5_days(ticker):
+    '''Retrieve data spanning 5 days in 1 day increments'''
+
+    data = retrieve_historical(ticker, '5_days')
+
+    if not data:
+        return f'{ticker} not found', status.HTTP_400_BAD_REQUEST
+
+    return jsonify(data)
+
+
+@app.route('/<ticker>/historical/1_month')
```

Comment:
You could probably refactor those 5 routes into only `/<ticker>/historical/<duration>` and have a check that duration is valid before you pass it to `retrieve_historical()`

Reply:
[Resolved](https://github.com/BoringStocks/StonkScraper/pull/12/commits/5a50a171fff66bf2c7fe15e9a579086e56dd892e)